src/models/GitError.js

```diff
@@ -15,7 +16,7 @@ const messages = {
   ObjectTypeUnknownFail: `Object { oid } has unknown type "{ type }".`,
   ObjectTypeAssertionFail: `Object { oid } was anticipated to be a { expected } but it is a { type }. This is probably a bug deep in isomorphic-git!`,
   ObjectTypeAssertionInTreeFail: `Object { oid } in tree for "{ entrypath }" was an unexpected object type "{ type }".`,
-  ObjectTypeAssertionInRefFail: `{ ref } is not pointing to a "commit" object but a "{ type }" object.`,
+  ObjectTypeAssertionInRefFail: `{ ref } is not pointing to a "{ requiredType }" object but a "{ actualType }" object.`,
```

Comment:
I think changing `type` to `actualType` is a breaking change. For backwards compatibility lets keep `type` named `type`. and for consistancy with `ObjectTypeAssertionFail` let's use `expected` instead of `requiredType`.